### PR TITLE
fix: stale labeled arrow bounds cache after editing the label

### LINKED ISSUE #6721: [Bug] Wrong bounds for arrows with text
### Steps

1. Draw a horizontal 2-point arrow
2. Add one line of text to it
3. Edit it one more time, adding 1+ lines of text
4. Select the arrow with some other element
5. See that the selection bounds account only for 1 line of the arrow's text

[Screencast from 2023-06-27 16-18-11.webm](https://github.com/excalidraw/excalidraw/assets/45559664/d452e593-553f-4e16-9f6b-708a8738489f)

### PR DESCRIPTION
**Pull request by alex-kim-dev**
fix #6721

### CODE REVIEW

**Comment by dwelle** [comment]
thanks @alex-kim-dev!

@ad1992 do you see any concerns?

File: src/element/textWysiwyg.tsx [hunk 1]
```diff
@@ -541,6 +541,9 @@ export const textWysiwyg = ({
               id: element.id,
             }),
           });
+        } else if (isArrowElement(container)) {
```
Comment (ad1992) [inline src/element/textWysiwyg.tsx:544]: @alex-kim-dev can you add a test for this ?

File: src/element/textWysiwyg.test.tsx [hunk 2]
```diff
@@ -1509,4 +1510,33 @@ describe("textWysiwyg", () => {
       expect(text.text).toBe("Excalidraw");
     });
   });
+
+  it("should update labeled arrow bounds", async () => {
```
Comment (ad1992) [inline src/element/textWysiwyg.test.tsx:1514]: since the version is being updated now, that's what we should test in this test, can you update the test?

Comment (alex-kim-dev) [reply src/element/textWysiwyg.test.tsx:1514]: Oh ok

File: src/element/textWysiwyg.test.tsx [hunk 3]
```diff
@@ -1509,4 +1509,30 @@ describe("textWysiwyg", () => {
       expect(text.text).toBe("Excalidraw");
     });
   });
+
+  it("should update labeled arrow version", async () => {
```
Comment (ad1992) [inline src/element/textWysiwyg.test.tsx:1513]:
```suggestion
  it("should bump the version of labelled arrow when label updated", async () => {
```

**Review by ad1992 — APPROVED** [review]
LGTM

**Pull request merged**